WordPress/Classes/Services/RecentSitesService.swift

```diff
@@ -79,5 +79,6 @@ class RecentSitesService: NSObject {
             return
         }
         touch(site: url)
+        WP3DTouchShortcutCreator().createShortcutsIf3DTouchAvailable(true)
```

Comment:
Rather than call this directly here, I'd post a notification and handle it like the sign in one